myst_nbs/case_studies/conditional_autoregressive_priors.myst.md

```diff
@@ -0,0 +1,355 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.7
+kernelspec:
+  display_name: Python 3.10.5 ('pymc-dev-car-nb')
+  language: python
+  name: python3
+substitutions:
+  extra_dependencies: bambi seaborn
```

Comment:
It doesn't look like bambi is used. The libraries here are the ones that should be installed in addition to pip install pymc.